rules/windows/file_delete/file_delete_win_webserver_access_logs_deleted.yml

```diff
@@ -0,0 +1,22 @@
+title: WebServer Access Logs Deleted
+id: 3eb8c339-a765-48cc-a150-4364c04652bf
+status: experimental
+description: Detects the deletion of WebServer access logs which may indicte the attempt to destroy forensic evidence
+references:
+    - https://www.elastic.co/guide/en/security/current/webserver-access-logs-deleted.html
+author: Tim Rauch
+date: 2022/09/16
+logsource:
+    category: file_delete
+    product: windows
+tags:
+    - attack.defense_evasion
+    - attack.t1070
+detection:
+    selection:
+        FileName|startswith: 'C:\inetpub\logs\LogFiles\'
+        FileName|endswith: '.log'
+    condition: selection
+falsepositives:
+    - Unknown
```

Comment:
A good false-positive description for this might be 
- During uninstallation of the IIS service 
- During log rotation
These two seem like they would trigger some FP so we should mention them